github.go

```diff
@@ -68,6 +69,15 @@ func NewGitHubPullReqest(cli *github.Client, owner, repo string, pr int, sha str
 	}
 }
 
+func F() {
+	go func() {
+		var tick = time.Tick(1 * time.Second)
```

Comment:
**[staticcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
using time.Tick leaks the underlying ticker, consider using it only in endless functions, tests and the main package, and use time.NewTicker here (SA1015)